public/index.html

```diff
@@ -9,7 +9,7 @@
 	<link rel="manifest" href="%PUBLIC_URL%/manifest.json">
```

Comment:
I'm not sure how to deal with this, but in Chrome there's an error `Manifest: Line: 1, column: 1, Syntax error.`  manifest.json